error.js

```diff
@@ -52,4 +52,7 @@ error.QueryFileError = pgp.errors.QueryFileError
 error.QueryResultError = pgp.errors.QueryResultError
 error.ValidationError = ValidationError
 
+error.queryResultErrorNoData = { code: pgp.errors.queryResultErrorCode.noData }
+error.queryResultErrorMultiple = { code: pgp.errors.queryResultErrorCode.multiple }
```

Comment:
Not a fan of mixing error shapes in same namespace with error creators/throwers.
Maybe we ca put them under `error.shape`?